src/documentation/markdown/2017/lar-data-fields.md

```diff
@@ -0,0 +1,378 @@
+  - Varying values
+
+### [agency\_code](#agency_code)
+
+- **Description:** A financial institution's associated Agency Code
+- **Values:**
+  - 1 - Office of the Comptroller of the Currency (OCC)
+  - 2 - Federal Reserve System (FRB)
+  - 3 - Federal Deposit Insurance Corporation (FDIC)
+  - 4 - Office of Thrift Supervision (OTS)
+  - 5 - National Credit Union Administration (NCUA)
+  - 7 - Department of Housing and Urban Development (HUD)
+  - 9 - Consumer Financial Protection Bureau (CFPB)
+
+### [loan\_type](#loan_type)
+
+- **Description:** The type of covered loan or application
+- **Values:**
+  - 1 - Conventional (not insured or guaranteed by FHA, VA, RHS, or FSA)
+  - 2 - Federal Housing Administration insured (FHA)
+  - 3 - Veterans Affairs guaranteed (VA)
+  - 4 - USDA Rural Housing Service or Farm Service Agency guaranteed (RHS or FSA)
+
+### [property\_type](#property_type)
+
+- **Description:** Classification of dwelling
+- **Values:** 
+ - 1 - Single family with 1-4 units
+ - 2 - Manufactured housing
+ - 3 - Multifamily housing
+
+### [loan\_purpose](#loan_purpose)
+
+- **Description:** The purpose of covered loan or application
+- **Values:**
+  - 1 - Home purchase (one-to-four family)
+  - 2 - Home improvement (one-to-four family)
+  - 3 - Refinancing (home purchase or home improvement,
+ one-to-four family) 
+  - 4 - Multifamily dwelling (home purchase, home improvement, and refinancings)
+
+### [occupancy_type](#occupancy_type)
+
+- **Description:** Occupancy type for the dwelling
+- **Values:**
+  - 1 - Principal residence
+  - 2 - Second residence
+  - 3 - Investment property
+
+### [loan\_amount](#loan_amount)
+
+- **Description:** The amount of the covered loan, or the amount applied for
+- **Values:**
+  - Varying values  
+
+### [preapproval](#preapproval)
+
+- **Description:** Whether the covered loan or application involved a request for a preapproval of a home purchase loan under a preapproval program 
+- **Values:**
+  - 1 - Preapproval requested
+  - 2 - Preapproval not requested
+
+### [action\_taken](#action_taken)
+
+- **Description:** The action taken on the covered loan or application
+- **Values:**
+  - 1 - Loan originated
+  - 2 - Application approved but not accepted
+  - 3 - Application denied by financial institution 
+  - 4 - Application withdrawn by applicant
+  - 5 - File closed for incompleteness
+  - 6 - Loan purchased by your institution
+
+### [msa_md](#msa_md)
+
+- **Description:** The 5 digit MSA (metropolitan statistical area) or MD (metropolitan division) code. An MSA/MD is an area that has at least one urbanized area of 50,000 or more population.
+- **Values:**
+  - Varying values
+
+### [state\_code](#state_code)
+
+- **Description:** Two-letter state code
+- **Values:**
+  - Varying values
+
+### [county\_code](#county_code)
+
+- **Description:** State-county FIPS code
+- **Values:**
+  - Varying values
+
+### [census\_tract](#census_tract)
+
+- **Description:** 11 digit census tract number
```

Comment:
five digit Census Tract number with decimal. 
Example: 0109.02

Reply:
Removing the leading zero from the example to make it 5 digits + decimal.

Reply:
oh, nice catch!